src/site/content/en/mini-apps/mini-app-super-apps/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Mini apps and super apps
+authors:
+  - thomassteiner
+date: 2021-02-25
+# updated: 2021-02-25
+description: |
+  This chapter introduces the concept of super apps and presents the major players.
+tags:
+  - mini-apps
+---
+
+## Welcome to mini apps
+
+When you look at the applications on your phone, you probably have specific apps for specific tasks.
+You might have a banking app. You might have an app for buying public transit tickets. Likely you
+have an app for getting directions, etc. In this post, I introduce you to the concept of a different
+kind of app, namely mini apps, sometimes also referred to as mini programs or applets. I first
+provide some background on the various mini app platforms and their developer experience, and then
+focus on things the web can learn from mini apps. But before I start writing about mini apps, I
+first need to talk about super apps.
```

Comment:
```suggestion
You might have a banking app. You might have an app for buying public transit tickets. Likely you
have an app for getting directions, etc. In this post, I introduce you to the concept of a different
kind of app, namely mini apps, sometimes also referred to as mini programs or applets. I first
provide some background on the various mini app platforms and their developer experience, and then
focus on things the web can learn from mini apps. But before I start writing about mini apps, I
first need to talk about super apps.
```
```suggestion
You might have a banking app. You might have an app for buying public transit tickets. Likely you
have an app for getting directions, and many more specialized apps. This post introduces you to the 
concept of a different kind of apps—mini apps—sometimes also called mini programs or applets. 
You'll first learn about the background of various mini app platforms and their developer experience, and 
then focus on things the web can learn from mini apps. But before learning about mini apps, you first 
need to learn about super apps.
```

Reply:
Rephrased this to focus on the reader's POV, what they'll learn, rather than what you'll explain. We generally recommend avoiding writing in first person, so I'll suggest edits wherever it's not necessary.